featuretools/primitives/standard/aggregation/max_consecutive_positives.py

```diff
@@ -23,7 +23,10 @@ class MaxConsecutivePositives(AggregationPrimitive):
     """
 
     name = "max_consecutive_positives"
-    input_types = [ColumnSchema(semantic_tags={"numeric"})]
+    input_types = [
+        [ColumnSchema(logical_type=Integer)],
```

Comment:
So I guess this means this also doesn't work with the IntegerNullable logical type? We should create an issue to update these to work with all numeric types. It's not common for us to have primitives that work on only a subset of numeric types...

Reply:
Created
https://github.com/alteryx/featuretools/issues/2422
https://github.com/alteryx/featuretools/pull/2420